doc/nrf/ug_thread_tools.rst

```diff
@@ -37,22 +37,242 @@ Thread Border Router
+This solution is better when you are only interested in direct communication between your Border Router and the Thread network.
+For example, you can use the Docker container when you want to establish IP communication between an application running on Linux (such as Python Controller for Matter) and an application running on a Thread node.
+
+If you go for the Docker container option, make sure to :ref:`configure radio co-processor <ug_thread_tools_tbr_rcp>` first.
+
+To install and configure OpenThread Border Router on any Linux-based operating system using the Docker container, complete the following steps:
+
+#. Install the Docker daemon:
+
+   .. code-block:: console
+
+      sudo apt update && sudo apt install docker.io
+
+#. Start the Docker daemon:
+
+   .. code-block:: console
+
+      sudo systemctl start docker
+
+#. Create an IPv6 network for the OpenThread Border Router container in Docker:
+
+   .. code-block:: console
+
+      sudo docker network create --ipv6 --subnet fd11:db8:1::/64 -o com.docker.network.bridge.name=otbr0 otbr
+
+#. Download the latest version of the OpenThread Border Router Docker image by running the following command:
+
+   .. code-block:: console
+
+      docker pull openthread/otbr
+
+#. Connect the radio co-processor that you configured in :ref:`ug_thread_tools_tbr_rcp` to the Border Router device.
+#. Start the OpenThread Border Router container using the following command (in the last line, replace */dev/ttyACM0* with the device node name of the OpenThread radio co-processor):
+
+   .. parsed-literal::
+      :class: highlight
+
+      sudo docker run -it --rm --privileged --name otbr --network otbr -p 8080:80 \
+      --sysctl "net.ipv6.conf.all.disable_ipv6=0 net.ipv4.conf.all.forwarding=1 net.ipv6.conf.all.forwarding=1" \
+      --volume */dev/ttyACM0*:/dev/radio openthread/otbr --radio-url spinel+hdlc+uart:///dev/radio
+
+#. Form the Thread network using one of the following options:
+
+   * Follow the instruction in the `OpenThread Border Router Codelab tutorial step 2`_.
+   * Open the ``http://*localhost*:8080/`` address in a web browser and choose :guilabel:`Form` from the menu.
+
+     .. note::
+        If you are using a Raspberry Pi without a screen, but you have a different device in the same network, you can start a web browser on that device and use the address of the Raspberry Pi instead of *localhost*.
+
+#. To ensure that packets addressed to devices in the Thread network are routed through the OpenThread Border Router container in Docker,
+   run the following command with the On-Mesh Prefix that you set earlier for example, for ``fd11:22::/64`` prefix:
+
+   .. code-block:: console
+
+      sudo ip -6 route add fd11:22::/64 dev otbr0 via fd11:db8:1::2
+
+#. Check the status of the OpenThread Border Router by executing the following command:
+
+   .. code-block:: console
+
+      sudo docker exec -it otbr sh -c "sudo service otbr-agent status"
+      * otbr-agent is not running
```

Comment:
Shouldn't it be running?

Reply:
I removed the output of the commands, to align with the rest of the documentation.